packages/@aws-cdk/aws-batch/lib/job-definition.ts

```diff
@@ -77,6 +92,29 @@ export interface LogConfiguration {
   readonly secretOptions?: ExposedSecret[];
 }
 
+
+/**
+ * Fargate platform configuration
+ */
+export interface FargatePlatformConfiguration {
+  /**
+   * Fargate platform version
+   */
+  readonly platformVersion: ecs.FargatePlatformVersion
+}
+
+/**
+ * Fargate network configuration
+ */
+export interface NetworkConfiguration {
```

Comment:
Since `assignPublicIp` is the only property, i'd actually just get rid of this redundant nesting for now. 
If at some point more properties are introduced, we can bring it back.

Reply:
Done